mne/viz/tests/test_topomap.py

```diff
@@ -300,5 +302,16 @@ def test_plot_tfr_topomap():
     tfr._onselect(eclick, erelease, None, 'mean', None)
     plt.close('all')
 
+    # test plot_psds_topomap
+    info = raw.info.copy()
+    chan_inds = channel_indices_by_type(info)
+    info = pick_info(info, chan_inds['grad'][:5])
+
+    fig, axes = plt.subplots(nrows=2)
+    freqs = np.arange(0., 13.)
+    bands = [(4, 8, 'Theta'), (8, 12, 'Alpha')]
```

Comment:
> info = pick_info(info, chan_inds['grad'][:5])

@mmagnuski you could empirically find the number of channels at which the topomap call breaks. Maybe 2 would work and save more time.